keanu-python/tests/test_vertex.py

```diff
@@ -358,3 +364,18 @@ def test_java_vertex_to_python_vertex_persists_label() -> None:
     java_vertex = Gaussian(0., 1., label=label).unwrap()
     python_vertex = Vertex._from_java_vertex(java_vertex)
     assert python_vertex.get_label() == label
+
+
+def test_can_get_parents_and_children() -> None:
+
+    def labels_match(lhs, rhs) -> bool:
+        return [l.get_label() for l in lhs] == [r.get_label() for r in rhs]
+
+    parents = (Gaussian(0, 1, label="parent1"), Gaussian(0, 1, label="parent2"))
+    children = tuple(Gaussian(parents[0], parents[1], label="child%d" % i) for i in range(5))
```

Comment:
Nit: I have a bit of an irrational hatred of the modulo operator being used for string formatting in Python :).  In general for Python 3 I'd prefer to use either the .format() method or even better - have you ever come across the fstring stuff in Python?

For this, you should be able to just change to label=f"child{i}" and you'll get the correct string out.

Reply:
you just blew my mind

Reply:
switched to fString